sale_report_delivered/security/ir.model.access.csv

```diff
@@ -1,2 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_sale_report_delivered_manager,sale.report.delivered,model_sale_report_delivered,sales_team.group_sale_manager,1,1,1,1
```

Comment:
I'm thinking that you don't need this rule, and in fact, you should never grant delete permissions to `_auto = False` models.

Reply:
https://github.com/odoo/odoo/blob/32906d3ef32db9eb7e03e7ac768e3900e89b9c2c/addons/sale/security/ir.model.access.csv#L21-L22

Reply:
Look at this:

https://github.com/odoo/odoo/blob/18.0/addons/account/security/ir.model.access.csv#L7-L9

Seeing this one, you don't need as well modify not create permissions, just reading ones.

Reply:
That's 18.0 though. On 15.0, it' similar to yours (but only with reading):

https://github.com/odoo/odoo/blob/3a28e5b0adbb36bdb1155a6854cdfbe4e7f9b187/addons/account/security/ir.model.access.csv#L18-L20

I would bet that this is not a change in the framework, but a cleaning...

Is it worth trying? You tell me. I don't want to block this.

Reply:
In 18.0, we should look at this:
https://github.com/odoo/odoo/blob/18.0/addons/sale/security/ir.model.access.csv#L49

We normally recommend developers to do the following:
 - If this is the case in 15.0, it is always advisable to ‘mimic’ the version.
 - If it is wrong or can be improved, we can always do a PR to Odoo to change it and then adapt our modules.

If set as is in 18.0 with only this record:
`access_sale_report_salesman,sale.report,model_sale_report,sales_team.group_sale_sale_salesman,1,0,0,0,0,0`
should work without problems and with one less line, but I understand that we should not make people too confused so that they don't lose the spirit of contribution.

@pilarvargas-tecnativa , if you consider it appropriate, make the change, otherwise it's fine.

Reply:
I think it's good to follow the Odoo line but leave the read-only permission for the salesman group. Pushed change

Reply:
> If it is wrong or can be improved, we can always do a PR to Odoo to change it and then adapt our modules.

That's the question. It's not wrong. It's just that it seems not needed. Odoo is not going to accept that "improvement"/cleaning on stable versions, so if we detect that something can be done, the sooner, the better. If not, when migrating this module to 18, any contributor (being us or any other) won't probably do the change.

> but I understand that we should not make people too confused so that they don't lose the spirit of contribution.

This is not a random contributor. This is our team, and thus, we can set higher quality standards between us.